variables.tf

```diff
@@ -168,8 +168,9 @@ variable "kms_config" {
     crk_id           = string
     instance_id      = string
     private_endpoint = optional(bool, true) # defaults to true
+    account_id       = optional(string)     # To attach KMS instance from another account
   })
-  description = "Use to attach a Key Protect instance to the cluster"
+  description = "Use to attach a KMS instance to the cluster"
```

Comment:
Description "Use to attach a KMS instance to the cluster. if `account_id` is not provided, defaults to the account in use."

Reply:
Done.